plugin.py

```diff
@@ -93,6 +107,51 @@ def install_or_update(cls) -> None:
             shutil.rmtree(cls.basedir(), ignore_errors=True)
             raise
 
+    def on_pre_server_command(self, command: Mapping[str, Any], done_callback: Callable[[], None]) -> bool:
+
+        cargo_commands = []
+        for c in command["arguments"]:
+            if c["kind"] == "cargo":
+                cargo_commands.append(c)
+
+        if len(cargo_commands) == 0:
+            return False
+
+        window = sublime.active_window()
+        if window is None:
+            return False
+        view = window.active_view()
+        if not view:
+            return False
```

Comment:
Return `True` everywhere.

You could refactor it to not repeat `done_callback()` and `return True` everywhere.